js/views/conversation_view.js

```diff
@@ -1381,9 +1388,21 @@
         return;
       }
 
+      // If removable from server, we "Unsend" - otherwise "Delete"
+      const pluralSuffix = multiple ? 's' : '';
```

Comment:
i18n probably has pluralisation built in to it, we should be using that and not something like this.